### PR TITLE
search bar

### PR DESCRIPTION
**Pull request by YelyzavetaMolochynska**
[DEMO LINK](https://YelyzavetaMolochynska.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://YelyzavetaMolochynska.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hi! Good Job!  I left some comments, please check it

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+.bigsearch {
+  height: 68px;
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/style.css:23]: is not fixed

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/action_page.php"
+      method="get"
+      data-qa="big"
+    >
+      <div class="bigsearch">
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/index.html:19]: What function does this wrapper operate as?
You can replace your styles to <form> as a variant

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,45 @@
...
+        <input
+          class="input"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/index.html:25]: iss not fixed

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+.bigsearch :hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigsearch :focus {
+  outline: none;
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/style.css:58]: is not fixed

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+.smallsearch {
+  height: 40px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  position: relative;
+  display: flex;
+}
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/style.css:79]: is not fixed

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  font-weight: 500;
+}
+
+.loopsmall {
+  padding-top: 14px;
+  padding-left: 8px;
+  height: 11px;
+  position: absolute;
+  display: flex;
+}
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/style.css]: is not fixed

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hi!)
Pay attention to previous comments. I left them unresolved.
The fixes are not applied. 
Also, if you need any help - ask at fe_chat

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  line-height: 18px;
+}
+
+.inputsmall {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+::placeholder {
```
Comment (Viktor-Kost) [inline src/style.css:68]: Better not use attribute selector. Use class one

File: src/index.html [hunk 8]
```diff
@@ -11,6 +11,39 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/action_page.php"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="input search bigsearch"
```
Comment (Viktor-Kost) [inline src/index.html:20]:
```suggestion
        class="form__input"
```

File: src/index.html [hunk 9]
```diff
@@ -11,6 +11,39 @@
...
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      <img
+        src="/images/Search.svg"
+        alt="searchloop"
+        class="searchloop"
```
Comment (Viktor-Kost) [inline src/index.html:28]:
```suggestion
        class="form__loop"
```

File: src/index.html [hunk 10]
```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/action_page.php"
```
Comment (Viktor-Kost) [inline src/index.html:15]:
```suggestion
      class "form"
      action="/action_page.php"
```

File: src/index.html [hunk 11]
```diff
@@ -11,6 +11,39 @@
...
+        data-qa="keypress"
+      >
+      <img
+        src="/images/Search.svg"
+        alt="searchloop"
+        class="searchloop"
+      >
+    </form>
+    <form
+      action="/action_page.php"
```
Comment (Viktor-Kost) [inline src/index.html:32]: Here you add both base class and modifier. To get as base styles as modifier one
```suggestion
      class "form form--small"
      action="/action_page.php"
```

File: src/index.html [hunk 12]
```diff
@@ -11,6 +11,39 @@
...
+      method="get"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        class="inputsmall search smallsearch"
```
Comment (Viktor-Kost) [inline src/index.html:38]:
```suggestion
        class="form__input form__input--small"
```

File: src/index.html [hunk 13]
```diff
@@ -11,6 +11,39 @@
...
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <img
+        src="/images/Search.svg"
+        alt="searchloop"
+        class="loopsmall"
```
Comment (Viktor-Kost) [inline src/index.html:44]:
```suggestion
        class="form__loop form__loop--small"
```

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
```
Comment (Viktor-Kost) [inline src/style.css:21]: So here you will have a class `.form`, that have common styles for form 

_it's an example!_
```
.form {
  margin: 20px 0;
  border: solid 1px #e1e7ed;
  border-radius: 4px;
  height: 70px;
  position: relative;
  margin-top: 20px;
}
```
And then you use modifier that have ONLY differences with base class form

```
.form--small {
  height: 42px;
  border-radius: 4px;
}
```

So in this way you follow DRY (Don't repeat yourself) principe 
Apply this principe all over the styles

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
I see you haven't fixed previous comments yet. 
In general you need to refactor you code.
This is a plan, and i left you some comment to know how to do it
1) add base classes to BOTH forms, like `form`, `form__input`, `form__loop`
2) add modifiers `--small` ONLY for the second form classes
3) collect all basic css rules at base class
4) add to modifiers only rules that make difference with base class

If you need any help - please ask at fe_chat

File: src/style.css (hunk 7 above)
Comment (YelyzavetaMolochynska) [reply src/style.css:68]: It was the part of the task, that we can add styles through pseudo-element.

File: src/style.css (hunk 14 above)
Comment (YelyzavetaMolochynska) [reply src/style.css:21]: Thank you!

File: src/style.css (hunk 7 above)
Comment (OlEzhA0) [reply src/style.css:61]: I think Viktor means you should to use pseudo-element like `.input::placeholder` (example)

File: src/style.css [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+.form__input {
+  height: 66px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: Avenir;
```
Comment (OlEzhA0) [inline src/style.css:39]: why do we need to use `font-family: Avenir` a lot of times? Add this to `html` classes

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! Seems like you are almost done! 
Please double check unresolved issues
if you have any questions ping us in `fe_chat`

File: src/index.html [hunk 16]
```diff
@@ -11,6 +11,40 @@
...
+      <input
+        type="text"
+        class="form__input form__input--small"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <img
+      src="/images/Search.svg"
+      alt="searchloop"
+      class="form__loop form__loop--small"
+      >
```
Comment (Viktor-Kost) [inline src/index.html:47]:
```suggestion
      <img
        src="/images/Search.svg"
        alt="searchloop"
        class="form__loop form__loop--small"
      >
```

**Review by Viktor-Kost — APPROVED** [review]
Looks great! 
Well done!

P.S. Just a little comment let or you

**Pull request closed without merging**